### PR TITLE
metrics: convert delta() usage to rate(), except for directional negative

### LINKED ISSUE #4489: `delta()` should not be used in counter
## Bug Report

According to https://prometheus.io/docs/prometheus/latest/querying/functions/#delta: _"delta should only be used with gauges"_.
But we have some counters that use the delta function which may cause the wrong results. These metrics can be found in `pd.json`.

### What did you see instead?

Like previous issue: https://github.com/tikv/pd/issues/3561

Another one is the region heartbeat in the test cluster:
Screen Shot 2021-12-22 at 5 10 43 PM

### What version of PD are you using (`pd-server -V`)?
master

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

Issue Number: Close #4489

### What is changed and how does it work?

We need to change the metrics for operator, etcd, and scheduler from delta() to rate(), and multiply by 60 to ensure consistency before and after the modification. 

For hot region and balance regions, we should not make changes because these metrics are directional and have positive or negative values.
```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
#### operator
- Before
image
image

- After
image
image

#### scheduler
- Before
image
image
- After 
image
image

#### etcd
- Before
image
- After
image

#### heartbeat
- Before
image
image
- After
image
image

### Release note

```release-note
None.
```

### CODE REVIEW

File: metrics/grafana/pd.json [hunk 1]
```diff
@@ -11369,7 +11369,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(delta(pd_scheduler_region_heartbeat{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", store=~\"$store\", type=\"report\", status=\"err\"}[1m])) by (address, store)",
+              "expr": "sum(rate(pd_scheduler_region_heartbeat{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", store=~\"$store\", type=\"report\", status=\"err\"}[1m])) by (address, store)",
```
Comment (CabinfeverB) [inline metrics/grafana/pd.json:11372]: need change L11279 also？

Comment (lhy1024) [reply metrics/grafana/pd.json:11372]: I think we don't change it. We have multiple 60 times in L11279, so we should use opm.

Comment (CabinfeverB) [reply metrics/grafana/pd.json:11372]: I mean that we can change it and don't multiply 60. So all expressions will use `ops`

File: metrics/grafana/pd.json [hunk 2]
```diff
@@ -2692,7 +2692,7 @@
           },
           "yaxes": [
             {
-              "format": "opm",
+              "format": "ops",
```
Comment (rleungx) [inline metrics/grafana/pd.json:2695]: Is it possible to use opm since the store limit is based on minutes?

Comment (CabinfeverB) [reply metrics/grafana/pd.json:2695]: I think it's both OK, but the units need to be the same for at least one plate.

**Review by CabinfeverB — APPROVED** [review]

File: metrics/grafana/pd.json [hunk 3]
```diff
@@ -11179,7 +11179,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(delta(pd_scheduler_region_heartbeat{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", store=~\"$store\", type=\"report\", status=\"ok\"}[1m])) by (address, store)",
+              "expr": "sum(rate(pd_scheduler_region_heartbeat{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", store=~\"$store\", type=\"report\", status=\"ok\"}[1m])*60) by (address, store)",
```
Comment (CabinfeverB) [inline metrics/grafana/pd.json:11182]: In TiKV side, the time window is 2m. Do we need keep same with TiKV?

**Review by rleungx — APPROVED** [review]

**Review by CabinfeverB — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**